projects/plugins/jetpack/modules/related-posts/class.related-posts-customize.php

```diff
@@ -112,7 +119,7 @@ function customize_register( $wp_customize ) {
 	 * @since 4.4.0
 	 */
 	public static function render_callback() {
-		echo Jetpack_RelatedPosts::init()->get_headline();
+		echo esc_html( Jetpack_RelatedPosts::init()->get_headline() );
```

Comment:
Since the headline includes some HTML (an H3 heading tag as well as an `em` tag), I don't think we can escape it like this here.

Reply:
There are a few other instances in this file where it flagged for escaping that seemed a bit difficult to pull off or we could likely ignore. Maybe I'll remove this and do it in a separate PR just for escaping to figure all those out.